docs/index.md

```diff
@@ -6,6 +6,16 @@ nav_order: 1
 
 # Symphony BDK Reference Documentation
 
+> [!NOTE]
+> BDK version 3.0: Major change !
+> 
+> The newly introduced BDK version 3.0 relies on Java 17 and SpringBoot 3. This is a major change that allows Symphony to continue to propose the latest security fixes following the end of support of Spring Boot 2 and also to keep up with the latest evolutions of Java.
+> Symphony will continue to support the BDK 2.0 for a period of 6 months and on a best effort basis.
```

Comment:
For the next 6 months Symphony will provide critical security fixes for the BDK 2.0 where possible